### PR TITLE
Added manual cloudformation deployment

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

https://deploy-preview-1635--logz-docs.netlify.app/shipping/security-sources/aws-security-hub.html#manual-cloudformation-deployment

https://deploy-preview-1635--logz-docs.netlify.app/shipping/log-sources/aws-cost-and-usage-report.html#manual-cloudformation-deployment

https://deploy-preview-1635--logz-docs.netlify.app/shipping/log-sources/kinesis.html#manual-cloudformation-deployment

https://deploy-preview-1635--logz-docs.netlify.app/shipping/log-sources/cloudwatch.html#manual-cloudformation-deployment

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 1]
```diff
@@ -88,4 +108,101 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+<div class="tasklist">
+
+##### Select the Logz.io AWS Security Hub extension
+
+1. Navigate to **CloudFormation > Registry > Public extensions**.
+2. Set **Extension type > Modules** and **Publisher > Third party**.
+3. Select **logzio::awsSecurityHub::collector::MODULE**.
+
+
+##### Activate the Logz.io Kinesis extension
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:138]:
```suggestion
##### Activate the Logz.io AWS Security Hub extension
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 2]
```diff
@@ -88,4 +108,101 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |  
+|---|---|---|
+| logzioOperationsToken | Your Logz.io account token. {% include log-shipping/log-shipping-token.html %} |
+| logzioListener | {% include log-shipping/listener-var.html %} | 
+| logzioLogLevel | Required log level.  |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:172]: I think it's worth mentioning the possible log levels that this integration allows, they are mentioned in the "automated deployment" tab

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 3]
```diff
@@ -88,4 +108,101 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+    }
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |  
+|---|---|---|
+| logzioOperationsToken | Your Logz.io account token. {% include log-shipping/log-shipping-token.html %} |
+| logzioListener | {% include log-shipping/listener-var.html %} |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:171]: In this integration, they'll need to insert not only listener, but an http/https prefix, and the compatible port. For example - https://listener.logz.io:8071.
You can see in the automated deployment tab how we described it there

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 4]
```diff
@@ -88,4 +108,101 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+        }
+    }
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |  
+|---|---|---|
+| logzioOperationsToken | Your Logz.io account token. {% include log-shipping/log-shipping-token.html %} |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:170]: There's no `<<LOG-SHIPPING-TOKEN>` in the code sample

**Review by mirii1994 — CHANGES_REQUESTED** [review]
Regarding all files -
1. Do you think maybe we should add either in the overview or in the tab itself, that the manual deployment uses Cloudwatch Public Registry?
2. I think it's worth mentioning that our Public Registry extensions are currently available only on the aws region us-east-1?
3. For the "Before you begin, you'll need" for the manual deployment - I don't think that they need an S3 bucket as a prerequisite.

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 5]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+##### Add your stack values to the configuration template
+
+```yaml
+{
+    "Resources": {
+        "MyModule": {
+            "Type": "logzio::awsSecurityHub::collector::MODULE",
+            "Properties": {
+                "logzioOperationsToken": "<<LOG-SHIPPING-TOKEN>>",
+                "logzioListener": "logzioListener",
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:164]:
```suggestion
                "logzioListener": "https://<<LISTENER-HOST>>:8071",
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 6]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+                "logzioLogLevel": "logzioLogLevel"
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:165]:
```suggestion
                "logzioLogLevel": "<<LOGZIO-LOG-LEVEL>>"
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 7]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |
+| --- | --- |
+| `logzioListener` | Your Logz.io [listener url](https://docs.logz.io/user-guide/accounts/account-region.html), followed by port `8070` or `8071`. For example, `https://listener.logz.io:8071` |
+| `logzioLogLevel` | Log level for the Lambda function. Defaults to `info`. Valid options are: `debug`, `info`, `warn`, `error`, `fatal`, `panic`. |
+| `logzioOperationsToken` | Your Logz.io [operations token](https://app.logz.io/#/dashboard/settings/general). |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:178]:
```suggestion
| `<<LOG-SHIPPING-TOKEN>>` | Your Logz.io [operations token](https://app.logz.io/#/dashboard/settings/general). |
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 8]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+    }
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |
+| --- | --- |
+| `logzioListener` | Your Logz.io [listener url](https://docs.logz.io/user-guide/accounts/account-region.html), followed by port `8070` or `8071`. For example, `https://listener.logz.io:8071` |
+| `logzioLogLevel` | Log level for the Lambda function. Defaults to `info`. Valid options are: `debug`, `info`, `warn`, `error`, `fatal`, `panic`. |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:177]:
```suggestion
| `<<LOGZIO-LOG-LEVEL>>` | Log level for the Lambda function. Defaults to `info`. Valid options are: `debug`, `info`, `warn`, `error`, `fatal`, `panic`. |
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 9]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+        }
+    }
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |
+| --- | --- |
+| `logzioListener` | Your Logz.io [listener url](https://docs.logz.io/user-guide/accounts/account-region.html), followed by port `8070` or `8071`. For example, `https://listener.logz.io:8071` |
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:176]:
```suggestion
| `<<LISTENER-HOST>>` | Your Logz.io [listener url](https://docs.logz.io/user-guide/accounts/account-region.html), followed by port `8070` or `8071`. For example, `https://listener.logz.io:8071` |
```

File: _source/logzio_collections/_security-sources/aws-security-hub.md [hunk 10]
```diff
@@ -88,4 +108,107 @@ Give the stack some time to deploy and the resources to get created. Once this i
...
+4. Paste the contents of the yaml file into the **Resources** section of the template as follows:
+
+   ```yaml
+   "Resources": {
+        "MyModule": {
+            "Type": "logzio::awsSecurityHub::collector::MODULE",
+            "Properties": {
+                "logzioOperationsToken": "<<LOG-SHIPPING-TOKEN>>",
+                "logzioListener": "https://<<LISTENER-HOST>>:8071",
+                "logzioLogLevel": "logzioLogLevel"
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/aws-security-hub.md:201]:
```suggestion
                "logzioLogLevel": "<<LOGZIO-LOG-LEVEL>>"
```

**Review by mirii1994 — CHANGES_REQUESTED** [review]

File: _source/logzio_collections/_log-sources/aws-cost-and-usage-report.md [hunk 11]
```diff
@@ -18,6 +18,17 @@ shipping-tags:
   - aws
 order: 830
 ---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Manual configuration](#manual-configuration)
```
Comment (ShiranAvidov) [inline _source/logzio_collections/_log-sources/aws-cost-and-usage-report.md:25]: The name of this tab should be "Automated CloudFormation deployment" or something like that (it's not a configuration)

File: _source/logzio_collections/_log-sources/aws-cost-and-usage-report.md [hunk 12]
```diff
@@ -18,6 +18,17 @@ shipping-tags:
...
+* [Deployment using a module](#module-deployment)
+{:.branching-tabs}
+
+<!-- tab:start -->
+<div id="manual-configuration">
+
 AWS Cost and Usage Reports function tracks your AWS usage and provides estimated charges associated with your account. This integration allows you to ship logs from your AWS Cost and Usage Reports to your Logz.io account.
```
Comment (ShiranAvidov) [inline _source/logzio_collections/_log-sources/aws-cost-and-usage-report.md:32]: I think this description and the info box should be in a separate "Overview" tab.

**Review by ShiranAvidov — CHANGES_REQUESTED** [review]
Looks good :) I added couple of comments to aws-cost-and-usage page.

**Review by mirii1994 — APPROVED** [review]
AWS Security Hub is good to go :)

**Pull request merged**